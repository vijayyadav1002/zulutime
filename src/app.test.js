import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from './app';

describe('<App/>', function () {
    it('should render App', function () {
        const component = shallow(<App/>);
        expect(toJson(component)).toMatchSnapshot();
    });
});