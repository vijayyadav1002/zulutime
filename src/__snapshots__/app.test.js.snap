// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> should render App 1`] = `
<Fragment>
  <WithStyles(CssBaseline) />
  <WithStyles(ForwardRef(Container))
    disableGutters={true}
    maxWidth="sm"
  >
    <WithStyles(ForwardRef(Typography))
      component="div"
      style={
        Object {
          "backgroundColor": "#ffea00",
          "height": "100vh",
        }
      }
    >
      <MaterialUIPickers />
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Container))>
</Fragment>
`;
